tests/app/notifications/test_process_notification.py

```diff
@@ -242,21 +242,24 @@ def test_persist_notification_increments_cache_if_key_exists(sample_template, sa
 
 
 @pytest.mark.parametrize((
-    'research_mode, requested_queue, notification_type, key_type, expected_queue, expected_task'
+    'research_mode, requested_queue, notification_type, key_type, reply_to_text, expected_queue, expected_task'
```

Comment:
The addition of the `reply_to_text` text on every line creates a huge diff. The real difference is new lines with a `reply_to_text` set, others have `None`